docs/src/examples/components/Popup/Variations/PopupExamplePosition.shorthand.tsx

```diff
@@ -1,145 +1,67 @@
 import React from 'react'
 import { Button, Grid, Popup } from '@stardust-ui/react'
 
+class PopupArrowExample extends React.Component<any, any> {
+  state = { popupOpen: false }
+
+  togglePopupState = () => {
+    this.setState(prev => ({ popupOpen: !prev.popupOpen }))
+  }
+
+  render = () => {
+    const { position, align, icon, padding } = this.props
+
+    const buttonStyles = { padding, height: '38px', minWidth: '64px' }
+
+    return (
+      <Popup
+        open={this.state.popupOpen}
+        align={align}
+        position={position}
+        trigger={
+          <Button
+            onClick={() => this.togglePopupState()}
+            onMouseLeave={() => this.setState({ popupOpen: false })}
```

Comment:
this is fine as those are docs examples, as well as because of what the official React doc is saying for that: 

https://reactjs.org/docs/faq-functions.html

![image](https://user-images.githubusercontent.com/9024564/46141374-09aa5780-c254-11e8-8a03-3338b2b00b6a.png)

------------

This is the logic that is provided on the client side, so it is safe for us to advertise it. A thing that I would like prevent is the following - and this seems to be a fair way to do that:

Reply:
that should not happen; clicking on another item should act like document click and close the opened ones; I don't think that we should add custom logic to examples for closing the popups (in order to avoid this case u mentioned) but it should be part of functionality

Reply:
as long as `Popup` is controlled component, it shouldn't - lets discuss, please

Reply:
please, see my comment where the problem (and reasons for the approach taken) is further explained: https://github.com/stardust-ui/react/pull/282#discussion_r221593766

Reply:
got it